backend/substrapp/tasks/tasks.py

```diff
@@ -797,11 +811,21 @@ def prepare_volumes(subtuple_directory, tuple_type, compute_plan_key, compute_pl
 
     # local volume for train like tuples in compute plan
     if compute_plan_key is not None:
-        volume_id = get_local_folder_name(compute_plan_key)
-        get_or_create_local_volume(volume_id)
+        local_path = path.join(subtuple_directory, 'local')
+
+        # Get CP local folder from local pvc
+        local_path_in_pvc = get_local_folder(compute_plan_key)
+        get_or_create_local_volume(get_local_folder_name(compute_plan_key))
+
+        # Copy content of local folder from pvc to local folder in subtuple directory
+        # It allows the cp subtuple to be retried with compromising the
+        # local data.
+        if os.path.exists(local_path_in_pvc):
+            distutils.dir_util.copy_tree(local_path_in_pvc, local_path)
+            logger.info(f'Content from {local_path_in_pvc} has been copied to {local_path}')
```

Comment:
Are we able to differentiate the step from the path?
This message (in `prepare_volumes`) is identical to the one in `do_task`: it would be nice to be able to distinguish them in the logs